frontend/packages/container-security/src/components/image-manifest-vuln.tsx

```diff
@@ -305,19 +311,136 @@ export const ImageManifestVulnPage: React.FC<ImageManifestVulnPageProps> = (prop
+              <span>Security Scan</span>
+            </Tooltip>
+          </div>
+        </div>
+        <div className="co-m-table-grid__body">
+          {props.pod.status.containerStatuses.map((status) => (
+            <div className="row" key={status.containerID}>
+              <div className="col-md-3">
+                <ContainerLink pod={props.pod} name={status.name} />
+              </div>
+              <div className="col-md-4 co-truncate co-nowrap co-select-to-copy">
+                {props.pod.spec.containers.find((c) => c.name === status.name).image}
+              </div>
+              <div className="col-md-3">
+                {props.loaded ? (
+                  withVuln(
+                    vulnFor(status),
+                    (vuln) => (
+                      <span style={{ display: 'flex', alignItems: 'center' }}>
+                        <SecurityIcon
+                          color={
+                            vulnPriority.find(({ title }) => vuln.status.highestSeverity === title)
+                              .color.value
+                          }
+                        />
+                        &nbsp;
+                        <ResourceLink
+                          kind={referenceForModel(ImageManifestVulnModel)}
+                          name={vuln.metadata.name}
+                          namespace={props.pod.metadata.namespace}
+                          title={vuln.metadata.uid}
+                          displayName={`${totalFor(
+                            vulnPriority.findKey(
+                              ({ title }) => vuln.status.highestSeverity === title,
+                            ),
+                          )(vuln)} ${vuln.status.highestSeverity}`}
+                          hideIcon
+                        />
+                      </span>
+                    ),
+                    () => (
+                      <span>
+                        <GreenCheckCircleIcon />
+                        &nbsp;No vulnerabilities found
+                      </span>
+                    ),
+                  )
+                ) : (
+                  <div>
+                    <Loading />
+                  </div>
+                )}
+              </div>
+            </div>
+          ))}
+        </div>
+      </div>
+    </div>
+  );
+};
+
+export const ImageManifestVulnPodTab: React.FC<ImageManifestVulnPodTabProps> = (props) => {
+  return (
+    <Firehose
+      resources={[
+        {
+          isList: true,
+          kind: referenceForModel(ImageManifestVulnModel),
+          namespace: props.match.params.ns,
+          selector: {
+            matchLabels: { [podKey(props.obj)]: 'true' },
+          },
+          prop: 'imageManifestVuln',
+        },
+      ]}
+    >
+      {/* FIXME(alecmerdler): Hack because `Firehose` injects props without TypeScript knowing about it */}
+      <ContainerVulnerabilities pod={props.obj} {...(props as any)} />
```

Comment:
you could use `watchK8sResource` hook instead of `Firehose` and avoid `props as any` that way.

Reply:
I don't see a `watchK8sResource` hook, just an function that is passed as a prop by `withDashboardResources`, which doesn't apply to this scenario.

Reply:
Sorry, I meant this one https://github.com/openshift/console/blob/master/frontend/public/components/utils/k8s-watch-hook.ts#L64